required/latex-lab/testfiles-table-luatex/table-013-longtable-hyperref.tlg

```diff
@@ -21,6 +22,7 @@ Completed box being shipped out [1]
 ...\kern62.0
 ...\kern-1.00374
 ...\pdfdest name{page.1} xyz
+...\pdfdest name{page.1} xyz
```

Comment:
Compared with the same-named .tlg created by pdftex, it seems in luatex page destinations are written twice. It this expected?